src/stories/30_Monetization/DoubleClick.html

```diff
@@ -15,10 +15,10 @@
 <html ⚡ lang="en">
 <head>
   <meta charset="utf-8">
-  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/DoubleClick/">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <title>amp-story-auto-ads</title>
+  <title>DoubleClick</title>
```

Comment:
Suggestion: use a title that better describes the content, e.g.: DoubleClick Ads for AMP Stories

Reply:
done.